src/main/proto/ga4gh/coverage_service.proto

```diff
@@ -0,0 +1,86 @@
+syntax = "proto3";
+
+package ga4gh;
+
+// The coverage service returns binned counts for on requested containers.
+// It is designed to provide intermediate scale views of the data for genome
+// browsers.
+service CoverageService {
+  // Gets a list of `Dataset` matching the search criteria.
+  //
+  // `POST /datasets/search` must accept a JSON version of
+  // `SearchDatasetsRequest` as the post body and will return a JSON
+  // version of `SearchDatasetsResponse`.
+  rpc Coverage(CoverageRequest)
+    returns (CoverageResponse) {
+      option (google.api.http) = {
+        post: "/v0.6.0a8/coverage"
+        body: "*"
+      };
+    };
+}
+
+// A Bin message provides the 
+message Bin {
+  // The number of features in this bin.
+  uint64 count = 1;
+
+  // An optional map of additional information about this bin.
+  map<string, google.protobuf.ListValue> info = 4;
+}
```

Comment:
I think the notion was that perhaps a server would want to return information about the bin beyond the count. I'll remove it and wait and see if there are cases around it.